src/notify/ToasterList.interface.ts

```diff
@@ -12,7 +12,7 @@ export type StateToast = {
   options: ToastOptions
   open: boolean
   id: number
-  timeout: number | null
+  timeout: number | null | undefined
```

Comment:
Why do we need to allow undefined ?

Reply:
Because `planClose` can now be [undefined](https://github.com/habx/ui-core/pull/2169/files#diff-ec290ded78d414993450a1a3fd58e953d503dffe5e6e306098a585d3c26b634eR52) and in [here](https://github.com/habx/ui-core/blob/f4b65e7fad29e588ad9a960d9b1e87036bd9da92/src/notify/ToasterList.tsx#L87) it is assigned to `timeout`